graph/multi/multi.go

```diff
@@ -0,0 +1,112 @@
+// Copyright ©2014 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package multi
+
+import "gonum.org/v1/gonum/graph"
+
+// Node here is a duplication of simple.Node
+// to avoid needing to import both packages.
+
+// Node is a simple graph node.
```

Comment:
Should it still state that Node is a *simple* node?

Reply:
This is simple as in a simple thing, not simple as in a simple graph. This is true in simple as well since there is no such thing as a simple node in graph theory.